pkg/acsengine/const.go

```diff
@@ -55,7 +55,7 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"hyperkube":    "hyperkube-amd64:v1.6.6",
 		"dashboard":    "kubernetes-dashboard-amd64:v1.6.0",
 		"exechealthz":  "exechealthz-amd64:1.2",
-		"addonresizer": "addon-resizer:1.6",
+		"addonresizer": "addon-resizer:1.7",
```

Comment:
Each component is independent except for DNS.

Sync'd with Jaice, and there is value of doing a separate PR for each one:
1. get separate testing on each one
2. if there is a security issue in one of the revisions, we only have to back out one change.